get-latest-tgf.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
```

Comment:
You can init a variable here like that

```suggestion
#!/usr/bin/env bash

TGF=${TGF_PATH:=/usr/local/bin}/tgf

```